src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -4501,6 +4501,13 @@ PropagateNodeWideObjectsCommandList(void)
 	/* collect all commands */
 	List *ddlCommands = NIL;
 
+	if (EnableCreateDatabasePropagation)
+	{
+		/* Get commands for database creation */
```

Comment:
we start single-line comments with a non-capital letter

```suggestion
		/* get commands for database creation */
```